### ISSUE TITLE
[FEATURE]: Improvement in External Links Format and UI

### ISSUE THREAD

**Issue by AlefiyaAbbas** [Lissy93/git-into-open-source#45]
### Feature Suggestion

Problem:
I've noticed that the external links in the current UI are not well-formatted, and I believe there's room for improvement in the user interface for a better user experience.

The presentation of external links could be enhanced to make them more visually distinct, which would improve overall readability. Below are some contributions that can be made:
1. The icons size could be bigger to make them more noticeable and user-friendly.
2. The links can be displayed in some structured format (eg: indexing, categorizing links belonging to same group, etc). 
3. The links could have a short description or summary provided so that the user can  find what they are looking for easily. 
4. The links can open in a new tab to avoid user from navigating away from the site completely.

In summary, it can look something as:

![image](https://github.com/Lissy93/git-into-open-source/assets/94435326/fb703dc5-f360-49ac-8ef4-7971f1a2a9c1)

### Would you be up for working on this?

Yes, but with support

### Pre-Submission Checklist

- [X] I have checked this issue has [not already been raised](https://github.com/Lissy93/git-into-open-source/issues?q=is%3Aissue)
- [X] I've searched the [docs](https://github.com/Lissy93/git-into-open-source/tree/main/.github) for an answer
- [X] I agree to follow thr [Code of Conduct](https://github.com/Lissy93/git-into-open-source/blob/main/.github/CODE_OF_CONDUCT.md)

**Comment by priyanshusharma80** [comment]
Hi @Lissy93, I'm on the way to making it more interactive & visually appealing, I Will PR soon :) & will definitely let you know if there is any need for help..

**Comment by AyushDhimanIITR** [comment]
Hey, Can i work on this issue?

**Comment by Lissy93** [comment]
The links are pulled from [`resources.yml`](https://github.com/Lissy93/git-into-open-source/blob/main/resources.yml#L59) - so if extra info is needed, then it'd be possible to add a `description` field to each of those items listed under the `resources` array
